frontend/src/lib/components/CommandBar/SearchBar.tsx

```diff
@@ -0,0 +1,21 @@
+import { useMountedLogic } from 'kea'
+
+import { searchBarLogic } from './searchBarLogic'
+
+import SearchInput from './SearchInput'
+import SearchResults from './SearchResults'
+import SearchTabs from './SearchTabs'
+
+const SearchBar = (): JSX.Element => {
+    useMountedLogic(searchBarLogic)
```

Comment:
What is this used for?

Reply:
This is used to trigger the `afterMount` hook in the logic to fetch results immediately when opening the search bar.